uikit/src/main/res/layout/layout_bank_transfer_payment.xml

```diff
@@ -7,7 +7,7 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
-    <android.support.design.widget.TabLayout
+    <com.google.android.material.tabs.TabLayout
```

Comment:
In the past, this component causing an issue when host app using a different android library with mid-SDK, let say mid-SDK use AndroidX & host-app using legacy android support, vice versa. Can you confirm it not making any crash issu?

Reply:
I think if our SDK using AndoidX it will be required for HostApp to migrate to Android as well.